code/game/objects/items/devices/scanners/health_analyzer.dm

```diff
@@ -0,0 +1,543 @@
+			var/dmgreport = "<span class='info ml-1'>General status:</span>\
+							<table class='ml-2'><tr><font face='Verdana'>\
+							<td style='width:7em;'><font color='#ff0000'><b>Damage:</b></font></td>\
+							<td style='width:5em;'><font color='#ff3333'><b>Brute</b></font></td>\
+							<td style='width:4em;'><font color='#ff9933'><b>Burn</b></font></td>\
+							<td style='width:4em;'><font color='#00cc66'><b>Toxin</b></font></td>\
+							<td style='width:8em;'><font color='#00cccc'><b>Suffocation</b></font></td></tr>\
+							<tr><td><font color='#ff3333'><b>Overall:</b></font></td>\
+							<td><font color='#ff3333'><b>[CEILING(brute_loss,1)]</b></font></td>\
+							<td><font color='#ff9933'><b>[CEILING(fire_loss,1)]</b></font></td>\
+							<td><font color='#00cc66'><b>[CEILING(tox_loss,1)]</b></font></td>\
+							<td><font color='#33ccff'><b>[CEILING(oxy_loss,1)]</b></font></td></tr>"
+
+			if(mode == SCANNER_VERBOSE)
+				for(var/obj/item/bodypart/limb as anything in damaged)
+					if(limb.bodytype & BODYTYPE_ROBOTIC)
+						dmgreport += "<tr><td><font color='#cc3333'>[capitalize(limb.name)]:</font></td>"
+					else
+						dmgreport += "<tr><td><font color='#cc3333'>[capitalize(limb.plaintext_zone)]:</font></td>"
+					dmgreport += "<td><font color='#cc3333'>[(limb.brute_dam > 0) ? "[CEILING(limb.brute_dam,1)]" : "0"]</font></td>"
+					dmgreport += "<td><font color='#ff9933'>[(limb.burn_dam > 0) ? "[CEILING(limb.burn_dam,1)]" : "0"]</font></td></tr>"
+			dmgreport += "</font></table>"
+			render_list += dmgreport // tables do not need extra linebreak
+		for(var/obj/item/bodypart/limb as anything in carbontarget.bodyparts)
+			for(var/obj/item/embed as anything in limb.embedded_objects)
+				render_list += "<span class='alert ml-1'>Embedded object: [embed] located in \the [limb.plaintext_zone]</span>\n"
+
+	if(ishuman(target))
+		var/mob/living/carbon/human/humantarget = target
+
+		// Organ damage, missing organs
+		if(humantarget.internal_organs && humantarget.internal_organs.len)
+			var/render = FALSE
+			var/toReport = "<span class='info ml-1'>Organs:</span>\
+				<table class='ml-2'><tr>\
+				<td style='width:6em;'><font color='#ff0000'><b>Organ:</b></font></td>\
+				[advanced ? "<td style='width:3em;'><font color='#ff0000'><b>Dmg</b></font></td>" : ""]\
+				<td style='width:12em;'><font color='#ff0000'><b>Status</b></font></td>"
+
+			for(var/obj/item/organ/organ as anything in humantarget.internal_organs)
+				var/status = organ.get_status_text()
+				if (status != "")
+					render = TRUE
+					toReport += "<tr><td><font color='#cc3333'>[organ.name]:</font></td>\
+						[advanced ? "<td><font color='#ff3333'>[CEILING(organ.damage,1)]</font></td>" : ""]\
+						<td>[status]</td></tr>"
+
+			var/missing_organs = list()
+			if(!humantarget.getorganslot(ORGAN_SLOT_BRAIN))
+				missing_organs += "brain"
+			if(/*!HAS_TRAIT_FROM(humantarget, TRAIT_NOBLOOD, SPECIES_TRAIT) &&*/ !humantarget.getorganslot(ORGAN_SLOT_HEART))
```

Comment:
you'd figure not checking the speciestraits would be problematic, but our organ code is in such a shit state it actually works out.

This will be a different story once #12821 is merged, but its fine for now